modules/ROOT/pages/error-handling.adoc

```diff
@@ -86,36 +86,44 @@ The XML for the default error handling example looks like this:
++
+The Studio console prints an error message. The message indicates that
 the default handler, `OnErrorPropagateHandler`, propagates a 404 error returned
 from the HTTP GET request. Mule identifies this error as HTTP:NOT_FOUND.
 +
 [source,txt,linenums]
 ----
-****************************************************************************
-
-ERROR 2019-08-05 21:49:38,323
-  [[error-handlers-normal].http.requester.HTTP_Request_configuration.04 SelectorRunner]
-  [event: ] org.mule.runtime.core.internal.exception.OnErrorPropagateHandler:
-****************************************************************************
+ERROR 2021-01-19 18:48:50,392
+  [[error-handlers-example].http.requester.HTTP_Request_configuration.11 SelectorRunner]
+  [processor: error-handlers-example/processors/0; event: 1d3baf11-5aa0-11eb-b96f-a483e7abe2b5] org.mule.runtime.core.internal.exception.OnErrorPropagateHandler:
+********************************************************************************
 Message               : HTTP GET on resource 'http://jsonplaceholder.typicode.com:80/somebadrequest' failed: not found (404).
+Element               : error-handlers-example/processors/0 @ error-handlers-example:error-handlers-example.xml:15 (Request)
+Element DSL           : <http:request method="GET" doc:name="Request" config-ref="HTTP_Request_configuration" path="/somebadrequest"></http:request>
 Error type            : HTTP:NOT_FOUND
-Element               : error-handlers-example/processors/0 @ error-handlers-normal:error-handlers-normal.xml:16 (Request)
-Element XML           : <http:request method="GET"
-                              doc:name="Request"
-                              doc:id="51e94e98-2a62-4fa7-98e0-942bed31ee11"
-                              config-ref="HTTP_Request_configuration"
-                              path="/somebadrequest">
-                        </http:request>
+FlowStack             : at error-handlers-example(error-handlers-example/processors/0 @ error-handlers-example:error-handlers-example.xml:15 (Request))
 
   (set debug level logging or '-Dmule.verbose.exceptions=true' for everything)
-****************************************************************************
+********************************************************************************
 ----
 +
-* A description of the HTTP:NOT_FOUND error appears in the browser tab you
+The error logs in the Studio console present the following information:
++
+** *Message*: The description of the error.
+** *Element*: The XML element that failed execution and caused the error.
+** *Element DSL*: The XML element that failed execution and caused the error, expressed in DSL.
+** *Error type*: The type of Mule error raised by the Mule app.
+** *Flowstack*: This field lists an entry for each flow that processed the Mule event and the processor that sent the event to the next flow. The last entry shows the flow and the processor that failed during the processing of the event.
```

Comment:
Can this one be made parallel to the others? ("Entries for...")

Can you remove the article from all? "Description of..." 

Remove terminating punctuation for non-sentences? (Or is the style to include if one is a full sentence? I can't remember.)